crates/milli/src/update/new/extract/searchable/mod.rs

```diff
@@ -27,3 +31,15 @@ pub fn match_searchable_field(
 
     selection
 }
+
+fn has_searchable_children<I, A>(field_name: &str, searchable: Option<I>) -> bool
+where
+    I: IntoIterator<Item = A>,
+    A: AsRef<str>,
+{
+    searchable.is_none_or(|fields| {
+        fields
+            .into_iter()
+            .any(|attr| match_field_legacy(attr.as_ref(), field_name) != PatternMatch::Parent)
```

Comment:
We are missing a test for that, specifically one with:
 ```yaml
serachableAttributes:
  old: ["meta.author"]
  new: ["meta.author", "meta.illustrator"]
 ```


```suggestion
            .any(|attr| match_field_legacy(attr.as_ref(), field_name) == PatternMatch::Parent)
```